components/discussion-form.js

```diff
@@ -57,20 +78,8 @@ export function DiscussionForm ({
         ...SubSelectInitial({ sub: item?.subName || sub?.name })
       }}
       schema={schema}
-      onSubmit={handleSubmit || (async ({ boost, ...values }) => {
-        const { error } = await upsertDiscussion({
-          variables: { sub: item?.subName || sub?.name, id: item?.id, boost: boost ? Number(boost) : undefined, ...values }
-        })
-        if (error) {
-          throw new Error({ message: error.toString() })
-        }
-
-        if (item) {
-          await router.push(`/items/${item.id}`)
-        } else {
-          const prefix = sub?.name ? `/~${sub.name}` : ''
-          await router.push(prefix + '/recent')
-        }
+      onSubmit={handleSubmit || (async ({ boost, cost, ...values }) => {
+        await anonUpsertDiscussion(cost, boost, values)
```

Comment:
The naming of these functions is confusing. It makes it seem like everything is anonymous.

Reply:
I renamed `useAnonymous` to `useInvoiceable` now. The `anon` suffix was replaced with `invoiceable`.

I first wanted to use `useInvoice` but that sounded too much like it would return an invoice like `useMe` returns the current user.